helm/oncall/values.yaml

```diff
@@ -365,9 +365,9 @@ env: {}
 ingress:
   enabled: true
   #  className: ""
-  annotations:
-    kubernetes.io/ingress.class: "nginx"
-    cert-manager.io/issuer: "letsencrypt-prod"
+  annotations: {}
+  #  kubernetes.io/ingress.class: "nginx"
+  #  cert-manager.io/issuer: "letsencrypt-prod"
```

Comment:
can you elaborate on this change? what about simply modifying `ingress.annotations` in your `values.yml` (or via `--set`) when these defaults are not applicable?

Reply:
I want to remove them because:
- I use oncall helm chart as a dependency in my deployment, so the default `values.yaml` file is take into account 
- I use `ingress.ClassName` key. So it enters in conflict with the ingress annotation, and kubernetes does not allow to have the annotation  `kubernetes.io/ingress.class` with `.spec.ingressClassName` in a manifest.
- I use a `clusterissuer` for certmanager. So I add the annotation  `cert-manager.io/cluster-issuer` and the other cert-manager annotation stands.  cert-manger does not deal with an `issuer` and a `cluster-issuer` annotations.

For the ingressClassName, you should also modify your template `templates/ingress-regular.yaml` as if you define a `ingress.className` key, then you have the annotation and `.spec.ingressClassName` that are defined, and the generated manifest does not work with latest kubernetes versions.